README.md

```diff
@@ -1,2 +1,127 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖MY Portfolio -ICT EXPO Conference page: <a name="about-project"></a>
+
+>ICT EXPO Conference page project is created using HTML and CSS Use JavaScript to manipulate DOM elements. Here on the project, I prepared Two page for both mobile and desktop users .
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- BUILT WITH -->
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Programming languages:   HTML, CSS3
+- Tools:  Git, GitHub, Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- Responsive
+- Mobile first
+- Popup Mobile and Desktop window
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Link to deployed project.
+
+- [Live Demo Link](https://damenam.github.io/ICT-EXPO-ETHIOPIA-Conference)
+- [videos Link](https://www.loom.com/share/04eed985fdfd4a2a83b10d548dd91ef0))
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Git and Code editor.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+``git clone `https://github.com/DamenaM/ICT-EXPO-ETHIOPIA-Conference.git`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 MIhertab Damena
+
+- GitHub: [DamenaM](https://github.com/DamenaM)
+- Twitter: [@DamenaM](https://twitter.com/DamenaM)
+- LinkedIn: [Mihertab Damena](linkedin.com/in/DamenaM)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/DamenaM/ICT-EXPO-ETHIOPIA-Conference.git).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project please give a ⭐️ to this repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thank [Kiju Shin GUI Designer Seoul, Korea).
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to include the correct name of the designer `Cindy Shin` in the acknowledgment section